### DISCUSSION TITLE
Facebook Login Provider Error

### DISCUSSION THREAD

**Discussion by gabrielemuscogiuri** [supabase/supabase#20448] [Questions]
Hello,

I am trying to integrate the Facebook provider into my project. I have followed your [documentation](https://supabase.com/docs/guides/auth/social-login/auth-facebook) very carefully, yet the login does not seem to be working. The error I receive after clicking on the "Login with Facebook" button is "the domain is not available" (within the domain https://www.facebook.com/dialog/oauth?XXXX
![Screenshot 2024-01-15 alle 20 07 46](https://github.com/supabase/supabase/assets/8657611/b7491129-3a19-4f08-af12-e6875a713406)
).

In short, when clicking on Login with Facebook, a redirect to Facebook's authentication is initiated, but it seems that the domain of my Supabase project is not recognized by Facebook. I have also tried entering the URL in both https and http in the valid domains space, without success.

No online guide seems to resolve this issue. Could someone provide support please?

Thank you very much in advance,
